packages/client/components/assignment-selections/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+  Button
+} from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import styled from 'styled-components'
+
+const DetailContainer = styled.div`
+  display: flex;
+  width: 100%;
```

Comment:
I think `width: 100%` is redundant here.

Reply:
> I think `width: 100%` is redundant here.

I checked it. The `width: 100%` makes the highlighted area from end to end. Without it, the highlighted area is just as big as text area.
![image](https://user-images.githubusercontent.com/16532416/70392117-563a0880-19aa-11ea-8362-e9aaa05a46b2.png)

Reply:
> I think `width: 100%` is redundant here.

I tried to use `Grid` here, but it doesn't achieve the same effects as my styled component.

Reply:
Are you sure? Flexbox should take the entire width when the direction is column. I feel this component is the same as `<Grid container direction="column" justify="space-between">`

Reply:
> Are you sure? Flexbox should take the entire width when the direction is column. I feel this component is the same as `<Grid container direction="column">`
> 
> Edit: I think `space-between` might also be unnecessary here.

You're right. Dunno why it didn't work for me. I probably forgot to put flex direction.